scripts/release/test/stage/setup/run.sh

```diff
@@ -16,6 +17,8 @@ scp -i ReleaseBuildInstanceKey.pem -o StrictHostKeyChecking=no -r pkg/* ubuntu@"
 aws s3 cp s3://algorand-devops-misc/tools/gnupg2.2.9_centos7_amd64.tar.bz2 .
 scp -i ReleaseBuildInstanceKey.pem -o StrictHostKeyChecking=no -r gnupg2.2.9_centos7_amd64.tar.bz2 ubuntu@"$INSTANCE":
 
-scp -i ReleaseBuildInstanceKey.pem -o StrictHostKeyChecking=no -r scripts/release/test/stage/setup/task.sh ubuntu@"$INSTANCE":
-ssh -i ReleaseBuildInstanceKey.pem -A ubuntu@"$INSTANCE" bash task.sh "$CHANNEL" "$RELEASE"
+#scp -i ReleaseBuildInstanceKey.pem -o StrictHostKeyChecking=no -r scripts/release/test/stage/setup/task.sh ubuntu@"$INSTANCE":
```

Comment:
Nit: you might want to delete old code..

Reply:
Ah, an oversight.  I won't remove it now, but I will in the future.